api/config/dev.exs

```diff
@@ -2,11 +2,11 @@ import Config
 
 # Configure your database
 config :api, Api.Repo,
-  username: "postgres",
-  password: "postgres",
-  hostname: "localhost",
-  database: "performance_manager",
-  port: "5432",
+  username: System.get_env("DB_USER"),
+  password: System.get_env("DB_PASSWORD"),
+  hostname: System.get_env("DB_HOST"),
+  database: System.get_env("DB_NAME"),
+  port: System.get_env("DB_PORT"),
```

Comment:
this change implies we'll only be able to run dev in a docker container until we figure out how we want to [handle env vars](https://app.asana.com/0/1189492770004753/1204390911340914/f) in the shell.

i needed to bring hostname over in order for the images to be able to find the locally run database.